src/status_im/ui/screens/accounts/recover/db.cljs

```diff
@@ -1,6 +1,7 @@
 (ns status-im.ui.screens.accounts.recover.db
-  (:require [cljs.spec.alpha :as s]
+  (:require [cljs.spec.alpha :as spec]
             status-im.utils.db))
 
-(s/def ::passphrase :global/not-empty-string)
-(s/def ::password :global/not-empty-string)
+(spec/def :account/passphrase :global/not-empty-string)
```

Comment:
these specs are using only for recovery, so you shouldn't change it and `:account/signing-phrase` is extra here, please revert this file, we are using `:accounts` only for app-db specs, these two specs are using in UI

Reply:
So, they don't really represent an account, this is more an `account-recovery-request` entity?

Reply:
https://github.com/status-im/status-react/blob/develop/src/status_im/ui/screens/accounts/recover/views.cljs#L45, this is just validators for input fields